catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliConstants.java

```diff
@@ -24,23 +25,25 @@
 
   public static final String STANAG_VERSION = "STANAG 4559";
 
-  public static final Set<String> CONTENT_STRINGS =
-      new HashSet<>(
-          CollectionUtils.collect(
-              Arrays.asList(NsiliProductType.values()),
-              (Object object) -> {
-                NsiliProductType type = (NsiliProductType) object;
-                return type.getSpecName();
-              }));
-
-  public static final Set<ContentType> CONTENT_TYPES =
-      new HashSet<>(
-          CollectionUtils.collect(
-              Arrays.asList(NsiliProductType.values()),
-              (Object object) -> {
-                NsiliProductType type = (NsiliProductType) object;
-                return new ContentTypeImpl(type.getSpecName(), STANAG_VERSION);
-              }));
+  private static final Set<String> CONTENT_STRINGS =
+      Collections.unmodifiableSet(
```

Comment:
If this is an unmodifiable set couldn't we just make this public?

Reply:
@josephthweatt good question. Apparently we can do that and SonarLint still likes the changes. However, I like hiding the details of variable names from other classes by using methods. I think that is still cleaner. Open to other thoughts on it